sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -649,7 +649,7 @@ private async Task<Response<RecognizeEntitiesResultCollection>> RecognizeEntitie
                     batchInput,
                     options.ModelVersion,
                     options.IncludeStatistics,
-                    _stringCodeUnit,
```

Comment:
This class a default value that needs to be deleted from here. See https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs#L29